doc-surrealdb_versioned_docs/version-latest/querying/graphql/surrealist.mdx

```diff
@@ -0,0 +1,137 @@
+---
+sidebar_position: 1
+sidebar_label: GraphQL via Surrealist 
+title: GraphQL via Surrealist | GraphQL
+description: In this section, you will explore interating with your SurrealDB instance using GraphQL queries using Surrealist, the official query editor for SurrealDB. Surrealist is a powerful tool that allows you to write, execute, and visualize GraphQL queries in real-time.
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# GraphQL via Surrealist 
+
+In this section, you will explore writing GraphQL queries in [Surrealist](https://surrealist.app/query). The GraphQL query view in Surrealist provides a rich set of features, including syntax highlighting, query validation, and query execution. You can see the results of your queries in JSON structure returned by GraphQL.
+
+## Getting Started
+
+Before you can start making queries, you need to start SurrealDB with the GraphQL module enabled. You can do this by setting the `SURREAL_EXPERIMENTAL_GRAPHQL` [environment variable](/docs/surrealdb/installation/running/env_vars) to `true` and starting a new instance of SurrealDB with the [`surreal start` command](/docs/surrealdb/cli/start).
+
+```bash
+SURREAL_EXPERIMENTAL_GRAPHQL=true surreal start --log debug --user root --password root
```

Comment:
Same here for the Windows command and possibly deleting / combining the second warning in the NOTE section just below.